tests/requirements-mlp.txt

```diff
@@ -2,3 +2,5 @@ chgnet==0.4.0
 mace-torch==0.3.9
 torch-dftd==0.5.1
 sevenn==0.10.3
+orb-models==4.1.0
+pynanoflann @ git+https://github.com/dwastberg/pynanoflann#egg=af434039ae14bedcbb838a7808924d6689274168
```

Comment:
We can keep the git link in `requirements-mlp.txt`. But can we remove the #egg=af434039ae14bedcbb838a7808924d6689274168?